src/griffe_warnings_deprecated/extension.py

```diff
@@ -15,8 +16,13 @@
 
 def _deprecated(obj: Class | Function) -> str | None:
     for decorator in obj.decorators:
-        if decorator.callable_path in _decorators:
-            return str(decorator.value).split("(", 1)[1].rstrip(")").rsplit(",", 1)[0].lstrip("f")[1:-1]
+        if decorator.callable_path in _decorators and isinstance(decorator.value, ExprCall):
+            first_arg = decorator.value.arguments[0]
+            try:
+                return ast.literal_eval(first_arg)  # type: ignore
```

Comment:
```suggestion
                return ast.literal_eval(first_arg)  # type: ignore[arg-type]
```

I suppose :slightly_smiling_face: